Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertTopography.cs

```diff
@@ -10,7 +10,7 @@ namespace Objects.Converter.Revit
 {
   public partial class ConverterRevit
   {
-    public ApplicationObject TopographyToNative(Topography speckleSurface)
+    public ApplicationObject TopographyToNative(Topography speckleSurface, bool keepXYDuplicates = true)
```

Comment:
Is there a reason why we should be "keeping XY duplicates" in the Revit Converter? Revit doesn't allow for multiple points to coincide in XY so this seems redundant.

Either we do allow for duplicates (and then Revit conversion will fail) or we clean up all meshes with duplicates always to ensure they succeed.

Modifying duplicate points will affect existing triangulation (see comment below)

Reply:
The 'keepDuplicates' parameter was introduces to keep the current behavior: create Revit Topography from incoming object, fail if there are duplicates, and default to DirectShape. This parameter is set to False only for GisTopography, because I know for sure the points will be duplicated multiple times (for coloring purposes). And as Revit Topography doesn't care about vertex color, they can be removed. BUT for the rest of cases I would still keep the vertices without cleaning (like now), so if there is a complex uncompatible mesh, it will fallback to DirectShape. 

This distinction is also a reason to separate GisTopography, but we can potentially find a better way